drools-wb-webapp/pom.xml

```diff
@@ -932,7 +932,12 @@
             <fileset>
               <directory>${basedir}</directory>
               <includes>
-                <include>.errai/**</include>
+                <include>src/main/webapp/org.drools.workbench.DroolsWorkbench/</include>
+                <include>src/main/webapp/WEB-INF/deploy/</include>
+                <include>src/main/webapp/WEB-INF/classes/</include>
+                <include>src/main/webapp/WEB-INF/lib/</include>
+                <include>**/gwt-unitCache/**</include>
+                <include>.errai/</include>
```

Comment:
I am most probably out of context here, but do we need to create this stuff under src/main/webapp? Would it be maybe possible to copy the stuff from src/main/webapp to target/ and then fiddle with that?